### ISSUE TITLE
Optional args/kwargs are handled incorrectly

### ISSUE THREAD

**Issue by SantjagoCorkez** [mjschultz/py-radix#32]
All those methods with `network=None, masklen=None, packed=None` signature are improperly handled via C-ext.

While parsing args/kwargs with `PyArg_ParseTupleAndKeywords()` declaring arg/kwarg's signature with `s` leads to fill the value with `Py_None` object while the arg was explicitly passed as `None` from python. This could be eliminated declaring an arg/kwarg with `z` - that leads a `NULL` value in C code instead of `Py_None`

I've prepared a PR https://github.com/mjschultz/py-radix/pull/31

**Comment by SantjagoCorkez** [comment]
Also a case that shows the bug:
```
tr = radix.Radix()
tr.add('127.0.0.1', 8, None)  # causes TypeError of packed parameter (not bytes-like)
```

**Comment by mjschultz** [comment]
Should be available via [`pip install py-radix==0.10.0`](https://pypi.org/project/py-radix/0.10.0/)
